setup.py

```diff
@@ -11,24 +11,26 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/arangoml/dgl-adapter",
-    keywords=["arangodb", "dgl", "adapter"],
+    keywords=["arangodb", "dgl", "deep graph library", "adapter"],
     packages=["adbdgl_adapter"],
     include_package_data=True,
-    python_requires=">=3.7",
+    python_requires=">=3.8",
     license="Apache Software License",
     install_requires=[
         "requests>=2.27.1",
+        "rich>=12.5.1",
+        "pandas>=1.3.5",
         "dgl>=0.6.1",
-        "torch>=1.10.2",
-        "python-arango>=7.4.1",
+        "torch>=1.12.0",
+        "python-arango==7.6.0",
```

Comment:
double checking if we want to bind instead of lock, as we currently lock in the PyG Adapter: https://github.com/arangoml/pyg-adapter/blob/master/setup.py#L23

Reply:
@aMahanna imo this should be change to allow to `7.X` to maintain compatibility while allowing users to upgrade `python-arango`.
```suggestion
        "python-arango~=7.6",
```